File: king-admin-java/src/main/java/com/oukingtim/mapper/SysUserRoleMapper.java
package com.oukingtim.mapper;

import com.baomidou.mybatisplus.mapper.BaseMapper;
import com.oukingtim.domain.SysUserRole;

import java.util.List;

/**
 * Created by oukingtim
 */
public interface SysUserRoleMapper extends BaseMapper<SysUserRole> {

    /**
     * 根据用户id查询角色id集合
     * @param userId
     * @return
     */
    List<String> selectRoleIdsByUserId(String userId);
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/mapper/SysUserRoleMapper.kt
package com.oukingtim.mapper

import com.baomidou.mybatisplus.mapper.BaseMapper
import com.oukingtim.domain.SysUserRole

/**
 * Created by oukingtim
 */
interface SysUserRoleMapper : BaseMapper<SysUserRole> {
    /**
     * 根据用户id查询角色id集合
     * @param userId
     * *
     * @return
     */
    fun selectRoleIdsByUserId(userId: String): List<String>
}